README.rst

```diff
@@ -87,107 +87,162 @@ Fletch
-* KWIVER_ENABLE_SPROKIT -- Enable the Stream Processing Toolkit
-* KWIVER_ENABLE_TESTS -- Build the unit tests
-* KWIVER_ENABLE_TOOLS -- Build the command line tools (e.g. plugin_explorer)
-* fletch_DIR -- Install directory for the Fletch support packages.
+============================= ====================================================================
+``CMAKE_BUILD_TYPE``          The compiler mode, usually Debug or Release
+``CMAKE_INSTALL_PREFIX``      The path to where you want the kwiver build products to install
+``KWIVER_ENABLE_ARROWS``      Enable algorithm implementation plugins
+``KWIVER_ENABLE_C_BINDINGS``  Whether to build the Vital C bindings
+``KWIVER_ENABLE_DOCS``        Turn on building the Doxygen documentation
+``KWIVER_ENABLE_LOG4CPLUS``   Enable log4cplus logger back end
+``KWIVER_ENABLE_PYTHON``      Enable the Vital Python bindings (requires KWIVER_ENABLE_C_BINDINGS)
+``KWIVER_ENABLE_SPROKIT``     Enable the Stream Processing Toolkit
+``KWIVER_ENABLE_TESTS``       Build the unit tests
+``KWIVER_ENABLE_TOOLS``       Build the command line tools (e.g. plugin_explorer)
+``fletch_DIR``                Install directory of a Fletch build.
+============================= ====================================================================
 
 There are many more options.  Specifically, there are numerous options
 for third-party projects prefixed with ``KWIVER_ENABLE_`` that enable
 building the Arrows plugins that depend on those projects.  When building
 with the support of Fletch_ (set ``fletch_DIR``) the enable options for
 packages built by Fletch should be turned on by default.
 
+The following sections will walk you through the basic options for a minimal kwiver build.
 
-Dependencies
-------------
+Basic CMake generation via command line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Vital has minimal required dependencies (only Eigen_).
-Sprokit additionally relies on Boost_.
-Arrows and Sprokit processes are structured so that
-the code that depends on an external package is in a directory with
-the major dependency name (e.g. vxl, ocv). The dependencies can be
-turned ON or OFF through CMake variables.
+Note, This assumes your fletch was build with python support (Turn OFF if not)
 
-.. _Boost: http://www.boost.org/
+You will also need to replace the fletch path with your own::
+
+cmake </path/to/kwiver/source> -Dfletch_DIR:PATH=<path/to/fletch/builds/install> -DCMAKE_BUILD_TYPE=Release -DKWIVER_ENABLE_ARROWS:BOOL=ON -DKWIVER_ENABLE_CERES:BOOL=ON -DKWIVER_ENABLE_C_BINDINGS:BOOL=ON -DKWIVER_ENABLE_LOG4CPLUS:BOOL=ON -DKWIVER_ENABLE_OPENCV:BOOL=ON -DKWIVER_ENABLE_PROCESSES:BOOL=ON -DKWIVER_ENABLE_PROJ:BOOL=ON -DKWIVER_ENABLE_PYTHON:BOOL=ON -DKWIVER_ENABLE_SPROKIT:BOOL=ON -DKWIVER_ENABLE_TESTS:BOOL=OFF -DKWIVER_ENABLE_TOOLS:BOOL=ON -DKWIVER_ENABLE_TRACK_ORACLE:BOOL=OFF -DKWIVER_ENABLE_VISCL:BOOL=OFF -DKWIVER_ENABLE_VXL:BOOL=ON -DKWIVER_USE_BUILD_TREE:BOOL=ON
```

Comment:
You don't really need to specify everything here, just things differ from the default.   Because of a recent change, the specific arrows (OpenCV, VXL, etc.) are now enabled by default if you build the dependency if Fletch.  So I would suggest a more minimal subset:

    cmake </path/to/kwiver/source> -Dfletch_DIR:PATH=<path/to/fletch/builds/install>
                                   -DCMAKE_BUILD_TYPE=Release
                                   -DKWIVER_ENABLE_ARROWS:BOOL=ON
                                   -DKWIVER_ENABLE_C_BINDINGS:BOOL=ON
                                   -DKWIVER_ENABLE_PYTHON:BOOL=ON
                                   -DKWIVER_ENABLE_LOG4CPLUS:BOOL=ON
                                   -DKWIVER_ENABLE_SPROKIT:BOOL=ON
                                   -DKWIVER_ENABLE_PROCESSES:BOOL=ON
                                   -DKWIVER_ENABLE_TESTS:BOOL=OFF
                                   -DKWIVER_ENABLE_TOOLS:BOOL=ON
                                   -DKWIVER_USE_BUILD_TREE:BOOL=ON